### PR TITLE
KWS Fix

### PR DESCRIPTION
**Pull request by MJ10**
* The KWS training script ran into errors due to some breaking changes in the BaseRNN class. This PR fixes the issue.
* FastGRNNCUDA sometimes fails assert of contiguous allocation of tensor (required by the CUDA cell). Added explicit contiguous allocation of tensor.
* Added a desktop demo for the KWS model, and updated the README accordingly.

### CODE REVIEW

**Comment by SachinG007** [comment]
@harsha-simhadri, I checked the PR. This can be merged.

File: examples/pytorch/FastCells/KWS-training/kws-demo.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    fastgrnn = create_model(config.model, num_filt, 35)
+    fastgrnn.load_state_dict(torch.load(fastgrnn_model_path, map_location=torch.device('cpu')))
+    fastgrnn.normalize(None, None)
+    pred = PredictionThread()
+    rec = RecordingThread()
+    
+    pred.start()
+    rec.start()
+    pred.join()
+    rec.join()
```
Comment (ShikharJ) [inline examples/pytorch/FastCells/KWS-training/kws-demo.py:211]: Missing newline here.

File: examples/pytorch/FastCells/KWS-training/README.md [hunk 2]
```diff
@@ -65,9 +65,21 @@ python examples/pytorch/FastCells/train_classifier.py \
 	--lr_min 0.0005 --lr_scheduler CosineAnnealingLR --lr_peaks 0
 ```
 Drop the `--rolling` and `--max_rolling_length` options if you are going to run inference on 1 second clips,
-and do not plan to stream data through the model without resettting.
+and do not plan to stream data through the model without resettting. `$MODEL_DIR` should be set to the output path of the model. The training script will generate
```
Comment (ShikharJ) [inline examples/pytorch/FastCells/KWS-training/README.md:68]: It should be `resetting` no?

Comment (MJ10) [reply examples/pytorch/FastCells/KWS-training/README.md]: missed that, thanks!

**Comment by harsha-simhadri** [comment]
> @harsha-simhadri, I checked the PR. This can be merged.

What about other scripts depending on FastGRNNCuda? Like the vision ones. Does this affect them>

**Comment by MJ10** [comment]
@harsha-simhadri the FastGRNNCUDA change won't affect other models, just ensures the input tensor is contiguous.

**Review by harsha-simhadri — APPROVED** [review]

**Pull request merged**